modules/update-preparing-forced-update-risk.adoc

```diff
@@ -0,0 +1,25 @@
+// Module included in the following assemblies:
+//
+// * updating/preparing_for_updates/updating-cluster-prepare.adoc
+
+:_content-type: CONCEPT
+[id="update-preparing-forced-update-risk"]
+= Assessing the risk of forced updates
+
+If there is no other way to update a cluster, you can "force" an update by using the `--force` option in the CLI, which allows you to forcefully upgrade the cluster even when upgrade release image validation fails and the cluster is reporting errors.
+
+The following text will be returned by using the `$ oc adm upgrade --help` command in the CLI:
+
+`The cluster may report that the upgrade should not be performed due to a content verification error or update
+precondition failures such as operators blocking upgrades. Do not upgrade to images that are not appropriately signed
+without understanding the risks of upgrading your cluster to untrusted code. If you must override this protection use
+the --force flag.`
```

Comment:
These are the output of 4.13 oc and earlier. With 4.14 oc, it returns

```
 Cluster-side guards include checks for release verification and upgradeable conditions. Again, it is usually best to give these conditions time to resolve, or to actively work to resolve them.  But if you decide to trigger the update regardless of these concerns, use --force, which is passed through to ClusterVersion's spec.desiredUpdate.force.
```

Reply:
updated :)